docs/tutorials/hardware/pwm_buzzer.mdx

```diff
@@ -0,0 +1,76 @@
+import BuzzerDiagram from "../../_images/hw_ff_buzzer.png";
```

Comment:
Not sure we typically use underscores (_) in the mdx file names? I think we typically stick to dashes (-).

This applies to 3 pwm files, I think.

Reply:
done.